_episodes/03-technical.md

```diff
@@ -101,6 +101,38 @@ are modifying.
 > screenshared.
 {: .challenge}
 
+## Configuring and Sync Forks
+
+Forks are often used in open source development on GitHub 
+to make changes to a project without affecting the original repository. 
+You must first configure a remote that points to the upstream repository in Git.
+Then, you can sync and fetch updates from or 
+submit changes to the original repository with pull requests. 
+
+To **configure a remote** for a fork, follow these commands:
+
+~~~
+$ git remote -v
+$ git remote add upstream https://github.com/ORIGINAL_OWNER/ORIGINAL_REPOSITORY.git
+$ git merge upstream/gh-pages
```

Comment:
> I have a `git fetch` command on line 125. Is that sufficient?

No, there needs to be a `git fetch` here, between `remote add` and `merge`.

Reply:
Would `git pull --rebase` be useful here as well? It avoids merge commits, which _only_ result from updating the local copy from the remote.

Reply:
``--rebase`` can create problem to novices.

Reply:
if you mean novice maintainers, I would argue that a clean history is helpful to themselves and to future maintainers and contributors as well. Besides, does a `pull --rebase` cause any problems when there are no conflicts in the `merge`d content itself?

Reply:
You could simply change `git merge` to `git rebase`. I think you can use pull (I don't ever use pull, so I'm reading from the manual here) as `git pull --rebase upstream gh-pages`.

Reply:
avoiding merge commits is nice but the GitHub web interface creates them anyways and we can't enforce/expect all maintainers to not include them in their repostiory history. I think it'd be ok to keep it "simple" by using `git merge upstream/gh-pages` (preceded by `git fetch` as pointed out by Rémi).